### DISCUSSION TITLE
Distributed Mode Setup in VM fails

### DISCUSSION THREAD

**Discussion by maiwandh** [Security-Onion-Solutions/securityonion#6193] [Unsupported Versions]
I successfully installed and configured a working standalone SO instance in VMware Workstation 16. Everything works as I expect.

Now I´m trying to setup a second installation of SO in distributed mode, as I want to use SO in production. I couldn´t find any deep or well explained guide for a distributed setup. Everything seems very high level till know.

Anyway. I tried to install it with two NICs. One with NAT and one in host-only. Alle the other machines from my vmware workstation are able to connect to the internet. Just this distributed setup gives me the following error in the /root/sosetup.log
![image](https://user-images.githubusercontent.com/43877044/141293841-2e02b96f-d668-4c4c-a47f-131e1381567c.png)

Quite clear, that the internet connection is missing. But can this relate to some misconfiguration or something which I missed?
I´m confused as alle the other machines from the same network are able to connect to the internet. The distributed manager node is only able to ping it´s gateway. 

P.S. My standalone machine ist hosted in a different virtual network. (192.168.245.0/24 -> IP: 192.168.245.254)
SO-distributed instance IP: 192.168.116.20

All the other machines from the network 192.168.116.0/24 are able to connect to the internet via NAT.

**Comment by dougburks** [comment]
Double-check your IP address, subnet mask, gateway, and DNS server settings. If you can ping the gateway but not resolve hostnames, that would seem to imply a DNS issue.

If it still doesn't work, try rebooting the host as occasionally VMware networks can get into weird states.

**Reply by maiwandh** [reply]
Update:

Initially the machine isn´t able to ping it´s own gateway. Only after starting the dhclient manually it´s able to ping it´s gateway but still not able to connect for example to google dns 8.8.8.8.

Same problem with Virtual Box. 

Does this behaviour change something to your answer before?

Additional question regarding to this topic:
Could it cause any problems if I install a distributed SO in a network where already a standalone SO is running?

**Reply by dougburks** [reply]
> Initially the machine isn´t able to ping it´s own gateway.

This seems to be a basic networking issue. Triple-check your IP address, subnet mask, and gateway settings.

> Only after starting the dhclient manually

Please keep in mind that we don't recommend DHCP for distributed deployments:
https://docs.securityonion.net/en/2.3/configuration.html?highlight=dhcp#configuration

> it´s able to ping it´s gateway but still not able to connect for example to google dns 8.8.8.8.

Perhaps your interfaces got reversed and so this is the host only network?

> Could it cause any problems if I install a distributed SO in a network where already a standalone SO is running?

Only if they are trying to use the same IP address.
